src/options.jl

```diff
@@ -3,19 +3,31 @@ struct Options
     print_empty::Bool
 end
 
+function Base.show(io::IO, options::Options)
```

Comment:
This should perhaps be 3 arg show (with `MIME"text/plain"`).

Reply:
Good point, fixed.